core/logger/orm.go

```diff
@@ -0,0 +1,42 @@
+package logger
+
+import (
+	"context"
+
+	"go.uber.org/zap/zapcore"
+	"gorm.io/gorm"
+)
+
+type ORM interface {
+	GetServiceLogLevel(serviceName string) (string, error)
+	SetServiceLogLevel(ctx context.Context, serviceName string, level zapcore.Level) error
+}
+
+type Orm struct {
```

Comment:
Shouldn't need to be public/exported -- convert to lowercase?

Reply:
doesn't look like a requirement, will convert to private